aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/parallel/ParallelStepTest.java

```diff
@@ -279,7 +279,7 @@ private void shouldHaveParallelStepsInTheOrder(String... expected) {
         for (Row row : t.getRows()) {
             ParallelLabelAction a = row.getNode().getAction(ParallelLabelAction.class);
```

Comment:
Actually this could now just be `ThreadNameAction`, permitting `ParallelLabelAction` to be made private.